web/main.scss

```diff
@@ -1,55 +1,62 @@
 $title-section-background-color: #202124;
 $body-background-color: #E9ECEF;
 
+$title-font: "Google Sans";
```

Comment:
So three font families: one for the title, one for the body, and one for the parameter input box.